app/views/categories/show.html.erb

```diff
@@ -0,0 +1,34 @@
+ <ul class="flex justify-between items-center p-4 bg-primary">
+        <li class="w-10"><%= link_to image_tag('backarrow.png', alt: 'Back'), categories_path, class: 'nav-link bg-primary' %></li>
+        <h2><strong class="text-white h-2"><%= @category.name %></strong></h2>
+        <li><h2><%= link_to 'Sign out', destroy_user_session_path, data: { turbo_method: :delete }, class: "nav-link text-white" %></h2></li>
+      </ul>
+
+    <div class="container">
+    <p style="color: green"><%= notice %></p>
+
+    <%= render @category %>
+    <div class="container-fluid">
+      
+        <% @records.each do |record| %>
+          <a href="<%= category_record_path(@category, record) %>" class="m-1">
+          <%= render "records/record", record: record %>
+          </a>
+        <% end %>
+      
+    </div>
+ 
+
+  <div class="flex flex-col space-y-2 mt-5">
+      <a href="<%= new_category_record_path(@category) %>" class="">
+        <button data-testid="new-record-button" class="btn btn-primary container">
+          New record
+        </button>
+      </a>
```

Comment:
- [ ] Please kindly ensure that the `New Record button` is designed according to the template. 

1. It should be fixed to the footer. 👍 
2. It should be designed in accordance to the given tempale. ℹ️ 

**Please take not that the same correction is applicable to `categories_index` page and `records_show` page**

## Your design
<img width="157" alt="Untitled" src="https://github.com/tajulafreen/Budget-app/assets/103386183/47da434b-2581-40f8-afbc-ee1ba5d04341">

## Given design
<img width="960" alt="Untitled" src="https://github.com/tajulafreen/Budget-app/assets/103386183/a83defdf-88c1-46ad-a67e-70d392069118">

## Design guideline
<img width="364" alt="Untitled" src="https://github.com/tajulafreen/Budget-app/assets/103386183/c7052996-d8aa-4a94-8cfb-6214d7f35b02">